.github/workflows/ci.yml

```diff
@@ -43,8 +43,21 @@ jobs:
       - name: Add wasm32 target
         run: rustup target add wasm32-unknown-unknown
 
-      - name: Build WASM
-        run: cargo build --lib --release --target wasm32-unknown-unknown
+      - name: Install Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20.11.0"
+
+      - name: Compile CF
+        run: cargo install -q worker-build && worker-build
+        working-directory: ./cloudflare
```

Comment:
we can specify step defaults on job level https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_iddefaults